### PR TITLE
LPS-198875 upgrade plexus-utils version 3.0.24

### PR DESCRIPTION
**Pull request by Seiphon**

### CODE REVIEW

**Comment by Seiphon** [comment]
ci:test:sf

**Comment by Seiphon** [comment]
ci:test:relevant

File: modules/util/portal-tools-soy-builder/src/main/resources/META-INF/maven/plugin.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
 				</parameter>
 			</parameters>
 			<configuration>
-				<_baseDir default-value="${project.basedir}" implementation="java.io.File"/>
```
Comment (drewbrokke) [inline modules/util/portal-tools-soy-builder/src/main/resources/META-INF/maven/plugin.xml]: @Seiphon were the changes to this file auto-generated?

**Review by drewbrokke — APPROVED** [review]
@seiphon just the one question about the plugin.xml file. Please see my inline comment. Otherwise this one looks good to go.

File: modules/util/portal-tools-soy-builder/src/main/resources/META-INF/maven/plugin.xml (hunk 1 above)
Comment (Seiphon) [reply modules/util/portal-tools-soy-builder/src/main/resources/META-INF/maven/plugin.xml]: > @Seiphon were the changes to this file auto-generated?

Yes, this changes are generated after executing "gw installCache"

Comment (drewbrokke) [reply modules/util/portal-tools-soy-builder/src/main/resources/META-INF/maven/plugin.xml]: Great, thanks for the clarification!

**Comment by drewbrokke** [comment]
ci:forward

**Comment by drewbrokke** [comment]
@Seiphon I think this is missing a fake gradle cache for the `com.liferay.portal.tools.soy.builder`

**Comment by Seiphon** [comment]
> @Seiphon I think this is missing a fake gradle cache for the `com.liferay.portal.tools.soy.builder`

Ooh sorry, I missed that, I will generated the cache and resend the pr

**Comment by Seiphon** [comment]
ci:test:sf

**Comment by Seiphon** [comment]
ci:test:relevant

**Comment by Seiphon** [comment]
Hey @drewbrokke , the sf issues is not related to my changes, I have sent another pr to fix it https://github.com/ling-alan-huang/liferay-portal/pull/2220

**Pull request closed without merging**